games/minimal/mods/test/hudscaletest.lua

```diff
@@ -0,0 +1,55 @@
+minetest.register_on_joinplayer(function(player)
+	minetest.after(1, function()
+		player:hud_add({
+			hud_elem_type = "text",
+			position = {x = 0.5, y = 0.4},
+			scale = {x = 100, y = 100},
+			text = "Small Hello World",
+			number = 0xFFFFFF,
+			alignment = {x = 1, y = 0},
+		})
```

Comment:
Two examples should be enough to demonstrate/test this feature.

Reply:
How about 3? 
![image](https://user-images.githubusercontent.com/28972859/82156003-8b863480-982d-11ea-9a47-58148ffc1252.png)

Reply:
Is the number of examples a concern when they're going to be removed before merge? Or did I miss an update?

****

But wow, the screenshot looks good... :)

Reply:
They're quite intrusive and only cover the one example. I'd prefer to remove them before merge